cmd/daemon/serve.go

```diff
@@ -69,6 +69,10 @@ func WithContext(ctx stdctx.Context) Option {
 	}
 }
 
+func init() {
+	graceful.DefaultShutdownTimeout = 120 * time.Second
```

Comment:
What is this for?

Reply:
When a long-running request (say bulk import) is in flight and the process is terminated (SIGINT for example), we want to wait for the request to complete before shutting down. The process will terminate as soon as all in-flight requests have concluded, but after 120 the latest.

The default here is 5 seconds, which I will hard-kill long-running requests leading to errors downstream.

Reply:
That makes sense - thank you!

Reply:
Can this lead to problems where the rollout is in flight and we are waiting for connections to finish, which may cause a downtime of ~2 minutes because of this?

Reply:
Why would waiting for connections to finish cause a downtime?

Reply:
I'm not sure how SQL migratinos behave but I believe that first migrations get executed, then the release is started. If the SQL migration is not online, we run into a downtime.

Reply:
But the old pods will still be serving traffic if the new release doesn't come up correctly, don't they?